src/routes/auth.js

```diff
@@ -25,7 +26,7 @@ const APIs = [
         route: '/organization/signup',
         method: 'post',
         multer: multer.single('coverImage'),
-        controller: authController.signUp,
+        controller: passport.authenticate,
```

Comment:
Why this line changed? controller here is authController.signUp

Reply:
My fault i shouldn't have changed it. I will fix this